src/contact.c

```diff
@@ -269,6 +280,24 @@ enum presence_status contact_presence(const struct contact *c)
 }
 
 
+enum sdp_dir contact_adir(const struct contact *c)
+{
+	if (!c)
+		return SDP_SENDRECV;
```

Comment:
Maybe return `SDP_INACTIVE`?

Reply:
For `sdp_dir` there is often the default value `SDP_SENDRECV`.

Reply:
But this is a error case not default.